grpc/creds/creds_test.go

```diff
@@ -0,0 +1,106 @@
+package creds
+
+import (
+	"crypto/rand"
+	"crypto/rsa"
+	"crypto/tls"
+	"crypto/x509"
+	"crypto/x509/pkix"
+	"math/big"
+	"net"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+	"time"
+
+	"github.com/letsencrypt/boulder/test"
+)
+
+func slowHandler(w http.ResponseWriter, r *http.Request) {
```

Comment:
It doesn't look like `slowHandler` is used anywhere. The timeout test doesn't reference it.